docs: Reorganize Subsections in the Intro to Contributing (#233)
open-sauced/docs: docs/contributing/introduction-to-contributing.md

3. Always review your code first. Run the project locally and test it before requesting a review.
4. Always communicate in the GitHub repository. Whether in the issue or the pull request, keeping the lines of communication open and visible to everyone on the team helps everyone around you.

## Issues

If you wish to work on an open issue, please comment on the issue with `.take`, and it will be assigned to you. If an issue is not assigned, it is assumed to be available for anyone to work on. Please assign yourself to an issue before working on it to avoid conflicts.

If you contribute to the project for the first time, you can search for issues with `good first issue` or `bug` labels in the repository.

In case you get stuck, feel free to ask for help in the [Discord](https://discord.gg/U2peSNf23P) server or [GitHub Discussions](https://github.com/orgs/open-sauced/discussions).

Please follow our [code of conduct](./code-of-conduct.md) in all your interactions with the project and its contributors.

> ## Commits

Author: Also, I'm debating between placing this section here or after the Pull Request section.

> When contributing to OpenSauced projects, you can use this command to follow our conventional commits:

Author: Assuming that the suggestion to move the last paragraph as an intro is approved, we might want to adjust this sentence as below:

```suggestion
So, when contributing to OpenSauced projects, please use this command to follow our conventional commits:
```

```shell
npm run push
```

> Using the `npm run push` command is an interactive replacement for `git commit`. It enforces the conventional commit specification for writing commit messages, making it easier for developers and maintainers to understand the changes made in a particular commit.

Author: ```suggestion
The `npm run push` command is an interactive replacement for `git commit`. It enforces the conventional commit specification for writing commit messages, making it easier for developers and maintainers to understand the changes made in a particular commit. After running this command, you can push your changes to the remote repo.
```

> Assuming you are dealing with code changes and you add them using `git add`, once you are ready to commit, there are two ways we can proceed: `git commit` or `npm run push`. The second method is preferred, as doing a subsequent `git push` and opening a PR would ensure the title conforms to our standards.

Author: I'm thinking to move this paragraph up as the intro paragraph for the Commits section and adjust the wordings a little bit. Any thoughts?

```suggestion
Assuming you are dealing with code changes and add them using `git add`. Once you are ready to commit, there are two ways you can proceed: `git commit` or `npm run push`. At OpenSauced, we prefer the second method, as doing a subsequent `git push` and opening a PR would ensure the title conforms to our standards.
```

Reviewer: Since it seems like there is a heavy preference to use `npm run push` it might be better to only present that option instead of saying "you can use git commit or npm run push"

The way I see it, new contributors to open source will probably just default to whatever the preferences that are laid out in the directions.
More regular contributors to open source will either default to the preference or just default to their preference of using git commit anyway.

Author: That's make sense. Thanks! 👍

- _[3 tips for getting your Pull Request reviewed on GitHub](https://youtu.be/cuMeC-eZJJ4)_

## Getting Started

### Setting Up Projects Locally

1. [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the repository you're working on from [github/open-sauced](https://github.com/open-sauced) to your own GitHub account.
2. Clone the forked repository to your local machine.
3. Check the project's README to determine how to run the project locally.

> For many of our repositories, you'll find the directions below work for running the project locally:
> 
> Run `npm ci` to install the dependencies and set up the project.
> 
> You can also use the shell commands below to get started once you have forked the repository:

Author: I need your thoughts. I'm thinking to make this a subsection (not sure about the title, though) or at least, adjust the wordings as below:

 ```suggestion
 #### Additional Information

For many of our repositories, you'll run `npm ci` to install the dependencies and set up the project locally.

You can also use the shell commands below to get started once you have forked the repository:
```

Reviewer: I would suggest adding a sentence explaining what `npm ci` is because a lot people will probably only be used to `npm i` if working on personal projects before they got involved with open source work.

Reviewer: I think the subsection works fine
